src/ue/nas/mm/messaging.cpp

```diff
@@ -73,7 +73,6 @@ static void RemoveCleartextIEs(nas::PlainMmMessage &msg, OctetString &&nasMsgCon
         regReq.micoIndication = std::nullopt;
         regReq.networkSlicingIndication = std::nullopt;
         regReq.mmCapability = std::nullopt;
-        regReq.requestedNSSAI = std::nullopt;
```

Comment:
Could you explain this remove?

Reply:
This is to prevent the UE from removing `Slice information`, before sending the NAS PDU. It is in reference to the comment [here](https://github.com/aligungr/UERANSIM/pull/731#discussion_r1822691504).